src/utils/hyprland.rs

```diff
@@ -13,10 +13,14 @@ pub struct HyprlandData {
 pub fn get_data() -> HyprlandData {
     HyprlandData {
         workspace: Workspace::get_active().unwrap().id,
-        window: if let Some(window) = Client::get_active().unwrap() {
-            window.title
-        } else {
-            String::default()
-        },
+        window: get_active_window_title(),
     }
 }
+
+/// Gets and returns the active window title
+fn get_active_window_title() -> String {
+    match Client::get_active().unwrap() {
```

Comment:
If `unwrap` fails then it should return an empty string, but I'll fix that myself at a later date.